apps/gp2-server/test/data-providers/working-group.data-provider.test.ts

```diff
@@ -77,5 +78,160 @@ describe('Working Group Data Provider', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const workingGroupDataObject =
+          parseWorkingGroupToDataObject(workingGroup);
+        const expected = getWorkingGroupDataObject();
+        expect(workingGroupDataObject).toEqual(expected);
+      });
+
+      test('the members in working group are parsed', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const workingGroupWithMembers = {
+          ...workingGroup,
+          flatData: {
+            ...workingGroup.flatData,
+            members: [
+              {
+                id: 'member-id',
+                role: 'Chair',
+                user: [
+                  {
+                    id: '42',
+                    created: '2021-01-01T00:00:00Z',
+                    lastModified: '2021-01-01T00:00:00Z',
+                    version: 1,
+                    flatData: {
+                      firstName: 'Tony',
+                      lastName: 'Stark',
+                      avatar: null,
+                    },
+                  },
+                ],
+              },
+            ],
+          },
+        };
+        const workingGroupDataObject = parseWorkingGroupToDataObject(
+          workingGroupWithMembers,
+        );
+        const expected = getWorkingGroupDataObject({
+          members: [
+            {
+              userId: '42',
+              role: 'Chair',
+              firstName: 'Tony',
+              lastName: 'Stark',
+            },
+          ],
+        });
+        expect(workingGroupDataObject).toEqual(expected);
+      });
+
+      test('avatar urls are added if available', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const workingGroupWithMembers = {
+          ...workingGroup,
+          flatData: {
+            ...workingGroup.flatData,
+            members: [
+              {
+                id: 'member-id',
+                role: 'Chair',
+                user: [
+                  {
+                    id: '42',
+                    created: '2021-01-01T00:00:00Z',
+                    lastModified: '2021-01-01T00:00:00Z',
+                    version: 1,
+                    flatData: {
+                      firstName: 'Tony',
+                      lastName: 'Stark',
+                      avatar: [{ id: 'avatar-id' }],
+                    },
+                  },
+                ],
+              },
+            ],
+          },
+        };
+        const workingGroupDataObject = parseWorkingGroupToDataObject(
+          workingGroupWithMembers,
+        );
+        const expected = getWorkingGroupDataObject({
+          members: [
+            {
+              userId: '42',
+              role: 'Chair',
+              firstName: 'Tony',
+              lastName: 'Stark',
+              avatarUrl: `${baseUrl}/api/assets/${appName}/avatar-id`,
+            },
+          ],
+        });
+        expect(workingGroupDataObject).toEqual(expected);
```

Comment:
same as above - not only is it difficult to understand what the input really is supposed to be - you are comparing the result to a large expectations where in fact you are just interested in the avatar

Reply:
done